### ISSUE TITLE
Show custom screen when user accept the call from lock screen.

### ISSUE THREAD

**Issue by ali7234** [masashi-sutou/flutter_ios_voip_kit#17]
@masashi-sutou First of all, thanks for this package. The issue I am facing is I have to navigate to the app when user accept the call from the lock screen. But in the current scenario, app stays in the native iOS Ui. Actually, I need to authenticate the app, then initiate the call with Agora.

**Comment by masashi-sutou** [comment]
@ali7234 

> But in the current scenario, app stays in the native iOS Ui.

- Tap your app icon on the lock screen like below image. Does this solve the problem?

![masashi-sutou_flutter_ios_voip_kit__One-to-one_video_call_using_CallKit_and_PushKit_with_flutter_iOS_app_](https://user-images.githubusercontent.com/6649643/95283026-b41ed180-0895-11eb-9480-de5588ebfb49.png)

**Comment by ali7234** [comment]
@masashi-sutou you are right, but the requirement is to directly navigate to the app when user unlock the device. Is there any way to do that in iOS CallKit.

**Comment by masashi-sutou** [comment]
> you are right, but the requirement is to directly navigate to the app when user unlock the device. Is there any way to do that in iOS CallKit.


@ali7234 

- I don't know how to directly open app when unlocking.

**Comment by rekonvald** [comment]
@ali7234  Hi! Have you solved this issue?  I have to get to app directly also...

**Comment by talwarabhimanyu** [comment]
Hi @ali7234 May I ask if you were able to solve this issue? It seems this is crucial for an intuitive calling interface.

**Comment by ali7234** [comment]
I do R&D but unable to find any solution or workaround of this issue at that time.

**Comment by joaquin03** [comment]
I need the same feature, to redirect to the app when accepting the call

**Comment by JayPerfetto** [comment]
+1
